pymess/utils/migrations.py

```diff
@@ -24,7 +24,9 @@ def __call__(self, apps, schema_editor):
         if self.template_slugs:
             email_template_qs = email_template_qs.filter(slug__in=self.template_slugs)
         email_template_list = list(email_template_qs)
+        if self.template_slugs:
+            assert len(email_template_list) == len(self.template_slugs), \
+                "The count of email templates given does not match the count of templates found"
```

Comment:
I don't like it. Which templates was not found? It can be hard to find the right one.

Reply:
Do you think there's a valid usage of `SyncEmailTemplates` when `len(email_template_list) == len(self.template_slugs)` is not True?

Reply:
Yes but you should return which slug was not found...

Reply:
ok. I have changed that:
```
...
File "/usr/local/lib/python3.9/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/usr/local/lib/python3.9/site-packages/pymess/utils/migrations.py", line 29, in __call__
    assert not email_templates_not_found, f"Email templates {email_templates_not_found} were not found."
AssertionError: Email templates {'non-existent'} were not found.
```